Mod Menu Helper/assets/modmenu/lang/el_gr.json

```diff
@@ -13,9 +13,10 @@
   "modmenu.summaryTranslation.citresewn": "Τα προσαρμοσμένα αντικείμενα του Optifine\n§9✎§8 Άνοιγμα/Κλέισιμο, \"σπασμένα\" πακέτα, ...",
   "modmenu.summaryTranslation.colormatic": "Τα προσαρμοσμένα χρώματα του Optifine\n§9✎§8 Καθαρός ουρανός/κενό, απαλότερο φως",
   "modmenu.summaryTranslation.continuity": "Οι ενωμένοι κύβοι του Optifine\n§9✎§8 Άνοιγμα/Κλέισιμο, μέθοδος, πακέτα πόρων",
-  "modmenu.summaryTranslation.cullleaves": "Τα \"βελτιωμένα δέντρα\"του OptiFine\n§9✎§8 Άνοιγμα/Κλέισιμο, πακέτο πόρων",
+  "modmenu.summaryTranslation.cull-less-leaves": "Γρηγορότερα φύλλα δέντρων\n§9✎§8 Άνοιγμα/Κλέισιμο, βάθος φύλλων",
   "modmenu.summaryTranslation.cem": "Οι προσαρμοσμένες οντότητες του Optifine\n§9✎§8 Υποστήρηξη Optifine, διόρθωση μοντέλων",
   "modmenu.summaryTranslation.dcch": "Παραμονή ιστορικού συνομιλίας ανά κόσμους\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.debugify": "Διορθώση προβλημάτων, κλείσιμο αποστολής διαγνωστικών \n§9✎§8 Δείτε το FO wiki για περ. πληρ.",
```

Comment:
![pilt](https://user-images.githubusercontent.com/8611110/167477806-11ccbab4-10ba-4ea3-8b7f-d6367f144c11.png)

Reply:
Should probably be resolved (?)